CHANGELOG.md

```diff
@@ -5,6 +5,31 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [v3.37.0](https://github.com/buildkite/agent/compare/v3.36.1...v3.37.0) (2022-07-06)
+
+### Added
+
+* Agent metadata includes `arch` (e.g. `arch=amd64`) alongside `hostname` and `os` [#1691](https://github.com/buildkite/agent/pull/1691) ([moskyb](https://github.com/moskyb))
+* Allow forcing clean checkout of plugins [#1636](https://github.com/buildkite/agent/pull/1636) ([toothbrush](https://github.com/toothbrush))
+
+### Fixed
+
+* Don't blow up when a hook exports an array [#1692](https://github.com/buildkite/agent/pull/1692) ([moskyb](https://github.com/moskyb))
```

Comment:
Perhaps "gracefully handle exported bash arrays in hooks" might be a bit more descriptive? This one is fully my fault for naming that PR so poorly :P

Reply:
👍🏼